utils/polus-imagej-util/generate.py

```diff
@@ -1,68 +1,86 @@
 import os
-import jpype
+import json
+#import jpype
 import shutil
-import classes.populate as cp
+#import classes.populate as cp
 from pathlib import Path
+import argparse
 
 
 """This file uses the classes in populate.py and cookiecutter to automatically 
 parse the ImageJ ops help and create plugins"""
 
 if __name__ == '__main__':
 
-    print('Starting JVM and parsing ops help\n')
-    
-    # Populate ops by parsing the imagej operations help
-    populater = cp.Populate()
-    
-    print('Building json templates\n')
-    
-    # Get the current working directory
-    cwd = Path(os.getcwd())
-    
-    # Save a directory for the cookietin json files
-    cookietin_path = cwd.joinpath('utils/polus-imagej-util/cookietin')
-    
-    # Build the json dictionary to be passed to the cookiecutter module 
-    populater.build_json('Benjamin Houghton', 'benjamin.houghton@axleinfo.com', 'bthoughton', '0.2.0', cookietin_path)
-    
-    print('Shutting down JVM\n')
-    
-    # Remove the imagej instance
-    del populater._ij
-    
-    # Shut down JVM
-    jpype.shutdownJVM()
-    
-    print('Generating plugins with cookiecutter\n')
-    
+    parser = argparse.ArgumentParser(prog='main', description='Generate Plugins')
+
+    # Add command-line argument for plugin name, docker repo and version
+    parser.add_argument(
+        '--plugins', 
+        dest='plugins_to_generate', 
+        type=str, 
+        nargs="+", 
+        default=["a", "b"],
+        help='Plugins which will be generated', 
+        required=True
+    )
+
+    # Parse the arguments
+    args = parser.parse_args()
+    plugins_to_generate = args.plugins_to_generate
+
+    # print('Starting JVM and parsing ops help\n')
+
+    # # Populate ops by parsing the imagej operations help
+    # populater = cp.Populate()
+
+    # print('Building json templates\n')
```

Comment:
Commented out code should be removed unless we absolutely want to keep it. We should replace `print` with `logger.debug` so that we don't fall into a scenario where we are removing and adding print statements for debugging.

Reply:
Also, create a logger. You can set the logger up to dump output to a file pretty easily.

Reply:
Debug logger added and removed commented lines of code